_spinnaker/account_management_api.md

```diff
@@ -0,0 +1,186 @@
+
+*There is a sample configuration in the manifests above.*
+
+
+    # mgmtapi.yml
+    storage:
+      github:
+        owner: armory-io
+        repo: spinnaker-remote-config
+        password: {redacted}
+        file: clouddriver-remote.yml
+
+When using the Github integration, you need to configure Clouddriver to use Github as well. Note, the `remote` profile corresponds to the `clouddriver-remote` file that the API manages. To enable remote configuration, add the snippet below to `~/.hal/{DEPLOYMENT_NAME}/profiles/spinnakerconfig.yml`.
+
+    spring:
+      profiles:
+        include: git,remote
+      cloud:
+        config:
+          server:
+            git:
+              uri: https://github.com/armory-io/spinnaker-remote-config 
+              password: {redacted}
+              username: doogie
+
+More details on configuring Spinnaker for remote configuration can be found [here.](https://www.spinnaker.io/setup/configuration/)
+
+## Usage
+
+The Account Management API works by hooking into the same backends as Spring’s Cloud Config - the core of Dynamic Accounts within Spinnaker. For example, if you are using Github as a backend for storing your account configuration, you can use the same backend with the Account Management API. Changes are reflected into Clouddriver as the Management API makes changes to the backing data.
+
+*Note -* *today,* *only Github is supported as a Management API backend.*
+
+Once configured, you can interact with the Account Management API using your favorite API clients. 
+
+Here’s an example using cURL:
+
+    $ curl -X POST http://mgmt-api-url/api/v1/accounts \
+      -d '{
+          "cloudProvider": "kubernetes",
+          "config": {
+                  "name": "added-from-api",
+                  "providerVersion": "v2",
+                  "kubeconfigContents": "",
+                  "metrics": false,
+                  "liveManifestCalls": false
+          }
+    }'
+
+**Verify Accounts**
+
+To verify that the account has been added successfully, use cURL to list accounts:
+
+    $ curl http://mgmt-api-url/api/v1/accounts | jq
+    [
+      {
+        "cloudProvider": "kubernetes",
+        "config": {
+          "kubeconfigContents": "",
+          "liveManifestCalls": false,
+          "metrics": false,
+          "name": "added-from-api",
+          "providerVersion": "v2"
+        }
+      }
+    ]
+
+The example includes an optional pipe of the output into JQ for readability.    
+
+**Delete Accounts**
+
+To delete an account with the API, call the `DELETE` route on a particular account by name. For example:
+
+    # curl -X DELETE http://mgmt-api-url/api/v1/accounts/added-by-api
+
+
+## Future Work
+
+Have functionality you want added to the Account Mangement API? [Contact us](https://www.armory.io/contact/) and to share your feedback and ideas.
```

Comment:
Do we need this sentence since the page is in the Armory Spinnaker docs? (rather than a blog post)